Sources/Basic/Process.swift

```diff
@@ -516,7 +516,7 @@ public final class Process: ObjectIdentifierProtocol {
         // Close the read end of the output pipe.
         close(fd)
         // Construct the output result.
-        return error.map(Result.init) ?? Result(out)
+        return error.map(Result.init) ?? Result.success(out)
```

Comment:
Just a nitpick, but I think this could just be `.success` since it's unambiguous.